.github/ISSUE_TEMPLATE/Bug-Report.yml

```diff
@@ -0,0 +1,66 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report for MCprep! We just need some info
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+    - type: textarea
+      id: steps-to-reproduce
+      attributes:
+        label: What steps are needed to produce the bug?
+        placeholder: Tell us what you did!
+        value: "Step 1: "
+      validations:
+        required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: MCprep Version
+      description: What version of MCprep are you using?
+      options:
+        - 3.3.1 (Default)
+        - 3.3.0 
+        - 3.2.5.1
+        - 3.2.5
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Blender Version
+      description: What version of Blender are you using?
```

Comment:
Is it possible to not hard code this? we will forever be out of date, rather just leave it a text entry field and give a sample format of entry. We might also want to allow something like "n/a" if it's a feature request (though that would be a different template I suppose).